party.py

```diff
@@ -11,6 +13,7 @@ class Party(metaclass=PoolMeta):
     documents = fields.One2Many('certification.document', 'party', 'Documents')
     valid_documents = fields.Function(
         fields.Boolean("Valid Documents"), 'get_valid_documents')
+    certification_not_available = fields.Boolean('Not Available')
```

Comment:
This boolean should not be at party.party but certification.document.

Reply:
8485b5fea39eceb754cbc90d617a3c7d3ad8effc